.gitignore

```diff
@@ -16,4 +16,5 @@ hera_pspec/_version.py
 ex.hdf5
 out.h5
 .coverage
-coverage.xml
\ No newline at end of file
+coverage.xml
+.DS_Store
```

Comment:
Is there a reason this changed? it seems unrelated to the PR

Reply:
Just added some stuff to avoid Mac's accidentally committing files we don't want.